components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -163,14 +163,17 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
         <div className="border-t border-b border-gray-200 dark:border-gray-800 mt-4 -mx-6 px-6 py-2">
           <p className="mt-1 mb-2 text-base">You already have running workspaces with the same context. You can open an existing one or open a new workspace.</p>
           <>
-            {result?.existingWorkspaces?.map(w =>
-              <a href={w.latestInstance?.ideUrl || gitpodHostUrl.with({ pathname: '/start/', hash: '#' + w.latestInstance?.workspaceId }).toString()} className="rounded-xl group hover:bg-gray-100 dark:hover:bg-gray-800 flex p-3 my-1">
-                <div className="w-full">
-                  <p className="text-base text-black dark:text-gray-100 font-bold">{w.workspace.id}</p>
-                  <p className="truncate" title={w.workspace.contextURL}>{w.workspace.contextURL}</p>
-                </div>
-              </a>
-            )}
+            {result?.existingWorkspaces?.map(w => {
+              const normalizedContextUrl = ContextURL.getNormalizedURL(w.workspace)?.toString() || "undefined";
```

Comment:
Note: This is one of the places where I was not 100% sure if we really want the `normalizedContextUrl`, but ultimately think we should always display it (except for debugging purposed in `/admin`, maybe).